Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -151,8 +161,16 @@
 		if dimASL(4)>=numel(jsonOut.GELabelingDuration)
 			warning('Labeling duration mismatch with GE private field.');
 			jsonOut.LabelingDuration = jsonOut.GELabelingDuration;
+		elseif dimASL(4)>=numel(unique(jsonOut.GELabelingDuration))
+			warning('Labeling duration mismatch with GE private field.');
+			labDurTemp = unique(jsonOut.GELabelingDuration);
```

Comment:
For consistency labelingDurationX throughout? `labelingDuration`, `labelingDurationTemp`, `labelingDurationGE`

Reply:
Yes, can fix. but that's a hell of a work - so will take me some time.

Reply:
GELabelingDuration is a DICOM tag. All our DICOM tags have vendor name as first. So it is better to keep the DICOM tag names consistent (also with dcm2nii style). So I changed to tempLabelingDuration - now it is all consistent.